Manual mocks example not working

I am trying to run the manual mocks example however I am getting this error. Node v0.10.38, using Jest v0.4.5 cloned directly from git repo.  I have tried the same test in Jest v0.3 and it doesn't work either. However it DOES work in v0.2.  From what I can see, it seems to be loading the real `fs` module rather than the mock.
```
> @ test /Users/markross/jest/examples/manual_mocks
> node ../../bin/jest.js

Using Jest CLI v0.4.5
 FAIL  __tests__/FileSummarizer-test.js (0.016s)
● FileSummarizer › listFilesInDirectorySync › it includes all files in the directory in the summary
  - TypeError: Object [object Object] has no method '__setMockFiles'
        at Spec.<anonymous> (/Users/markross/jest/examples/manual_mocks/__tests__/FileSummarizer-test.js:15:21)
        at jasmine.Block.execute (/Users/markross/jest/vendor/jasmine/jasmine-1.3.0.js:1065:17)
        at jasmine.Queue.next_ (/Users/markross/jest/vendor/jasmine/jasmine-1.3.0.js:2098:31)
        at null._onTimeout (/Users/markross/jest/vendor/jasmine/jasmine-1.3.0.js:2088:18)
        at Timer.listOnTimeout [as ontimeout] (timers.js:112:15)
  - Error: ENOENT, no such file or directory '/path/to'
      at Object.fs.readdirSync (fs.js:666:18)
      at Object.summarizeFilesInDirectorySync (/Users/markross/jest/examples/manual_mocks/FileSummarizer.js:4:13)
      at Spec.<anonymous> (/Users/markross/jest/examples/manual_mocks/__tests__/FileSummarizer-test.js:20:40)
      at [object Object].jasmine.Block.execute (/Users/markross/jest/vendor/jasmine/jasmine-1.3.0.js:1065:17)
      at [object Object].jasmine.Queue.next_ (/Users/markross/jest/vendor/jasmine/jasmine-1.3.0.js:2098:31)
      at [object Object]._onTimeout (/Users/markross/jest/vendor/jasmine/jasmine-1.3.0.js:2088:18)
      at Timer.listOnTimeout [as ontimeout] (timers.js:112:15)

1 test failed, 0 tests passed (1 total)
Run time: 0.948s
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0

--- a/examples/manual_mocks/__tests__/FileSummarizer-test.js
+++ b/examples/manual_mocks/__tests__/FileSummarizer-test.js
@@ -1,3 +1,4 @@
+jest.mock('fs');
 jest.dontMock('../FileSummarizer');
 
 describe('FileSummarizer', function() {

